### PR TITLE
Correcting some comments in geometries

### PR DESCRIPTION
**Pull request by josep-m-carbonell**
just to make it more clear

### CODE REVIEW

File: kratos/geometries/quadrilateral_3d_4.h [hunk 1]
```diff
@@ -419,41 +419,41 @@ template<class TPointType> class Quadrilateral3D4
         {
            area += temp[i] * integration_points[i].Weight();
         }
-        
+
         return area;
-        
+
 //         // 24/01/2014 - Massimo Petracca
```
Comment (loumalouomega) [inline kratos/geometries/quadrilateral_3d_4.h:425]: Maybe if this code is not going to be used we can remove it

**Review by loumalouomega — DISMISSED** [review]
Ok, just moving some commentaries

File: kratos/geometries/triangle_2d_6.h [hunk 2]
```diff
@@ -660,22 +660,21 @@ template<class TPointType> class Triangle2D6
...
-        NodesInFaces(0,0)=0;//face or master node
+        //face 1
+        NodesInFaces(0,0)=0;//contrary node to the face
         NodesInFaces(1,0)=1;
-        NodesInFaces(2,0)=4;
-        NodesInFaces(3,0)=2;
- 
-        NodesInFaces(0,1)=1;//face or master node
+        NodesInFaces(2,0)=2;
+        NodesInFaces(3,0)=4;//quadratic nodes after edge nodes
```
Comment (roigcarlo) [inline kratos/geometries/triangle_2d_6.h]: I think you are inverting these (same below)

File: kratos/geometries/quadrilateral_3d_4.h (hunk 1 above)
Comment (roigcarlo) [reply kratos/geometries/quadrilateral_3d_4.h:425]: ? I see no unused code

Comment (loumalouomega) [reply kratos/geometries/quadrilateral_3d_4.h:425]: 425 and following , deactivate the whitespace filter and you will see

**Review by roigcarlo — CHANGES_REQUESTED** [review]
Just blocking until I understand why some coords are swap

File: kratos/geometries/triangle_2d_6.h (hunk 2 above)
Comment (loumalouomega) [reply kratos/geometries/triangle_2d_6.h]: The commentary says itself "quadratic nodes after edge nodes"

Comment (josep-m-carbonell) [reply kratos/geometries/triangle_2d_6.h]: Yes there is an error here, let me correct it

Comment (loumalouomega) [reply kratos/geometries/triangle_2d_6.h]: In any case, @josep-m-carbonell can you create a test for the geometries in cpp as t is in the folder tests?

File: kratos/geometries/quadrilateral_3d_4.h (hunk 1 above)
Comment (roigcarlo) [reply kratos/geometries/quadrilateral_3d_4.h:425]: Ok, but maybe in another PR, has nothing to do with this one...

File: kratos/geometries/triangle_2d_6.h (hunk 2 above)
Comment (josep-m-carbonell) [reply kratos/geometries/triangle_2d_6.h]: Yes, that's right. It must be like this. First the  edges and then the other nodes, it is just a convention, I think only @AFranci was using it, he can say something about

**Comment by msandre** [comment]
The description/title of this PR is not consistent with the changes to the code. Such changes should be made in a separate PR.

File: kratos/geometries/triangle_2d_6.h (hunk 2 above)
Comment (josep-m-carbonell) [reply kratos/geometries/triangle_2d_6.h]: In fact I didn't want to merge this change in the PR. I forgot that this change was made. I will revert it.

**Review by roigcarlo — APPROVED** [review]
Good to me now :+1:

**Pull request merged**